runtime/lua/vim/_editor.lua

```diff
@@ -120,7 +119,30 @@ function vim._os_proc_children(ppid)
   return children
 end
 
--- TODO(ZyX-I): Create compatibility layer.
+-- Initializes v:parent (from server.c).
+function vim._server_init()
```

Comment:
Experimenting with defining core logic in Lua instead of C.

Will need to think more about where this kind of code lives, as we accumulate more of it.